test/built-ins/BigInt/asIntN/bigint-tobigint.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN type coercion for bigint parameter
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  2. Let bigint ? ToBigInt(bigint).
+features: [Symbol]
+---*/
+
+function MyError() {}
+
+assert.sameValue(BigInt.asIntN(3, Object(10n)), 2n);
```

Comment:
yes, I was nitpicking, requesting something like an explicit `new String` call instead of using `Object`. 

Anyway, that cannot happen with the BigInt ctor.